modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/quantity_selector/utils/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {BLOCK, DEFAULT, INPUT_STYLE} from './constants';
+
+export const generateOptions = ({
+	allowedQuantity = [],
+	maxQuantity = 1,
+	minQuantity = 1,
+	multipleQuantity = 1,
+}) => {
+	if (allowedQuantity.length > 0) {
+		return allowedQuantity;
+	}
+	else {
+		const quantitiesList = [];
+		for (let i = minQuantity; i <= maxQuantity; i++) {
+			quantitiesList.push(i * multipleQuantity);
+		}
+
+		return quantitiesList;
+	}
+};
+
+export const getInputStyle = (inputSize) => {
+	switch (inputSize) {
+		case BLOCK:
+			return INPUT_STYLE.BLOCK;
+		case DEFAULT:
+			return INPUT_STYLE.DEFAULT;
+		default:
+			// eslint-disable-next-line no-case-declarations
+			const n = parseInt(inputSize, 10);
+			// eslint-disable-next-line no-case-declarations
+			const width = n + 'px';
```

Comment:
Again, don't fight the linter. I expect it will stop complaining if you use block scope, as [the ESLint docs say](https://eslint.org/docs/rules/no-case-declarations):

eg.

```js
default: {
    const n = parseInt(inputSize, 10);
    const width = n + 'px';
}
```

But I would write that as a single variable anyway, because when we move to TypeScript it is going to complain about you "adding" a number and a string together, so maybe just `return` like you did in all the other cases:

```js
default:
    return {
        display: BLOCK,
        width: `${parseInt(inputSize, 10)}px`,
    };
```